packages/amplication-client/src/style/dark-css-variables.css

```diff
@@ -40,8 +40,8 @@
   --image-drop-excel: url("../assets/images/mono/drop-excel-dark.svg");
   --image-my-resources: url("../assets/images/mono/my-resources-dark.svg");
   --image-generating: url("../assets/images/mono/generating-dark.svg");
-  --image-create-service-wizard : url("../assets/images/mono/service-wizard-create.svg");
-
+  --image-create-service-wizard: url("../assets/images/mono/service-wizard-create.svg");
+  --image-commit-empty-state: url("../assets/images/mono/commit-empty-state.svg");
```

Comment:
please not that there are two flavor for each image - one for "dark" and another for "light". 
Altought we are not using the light theme any more, as long as we are keeping the infrastructure for themes, let's keep the new staff aligned. 

so, please rename the image to include the suffix "-dark" at the end, even if we don't have the light version